commands/wait.js

```diff
@@ -57,6 +57,7 @@ function * run (context, heroku) {
 module.exports = {
   topic: 'pg',
   command: 'wait',
+  description: 'blocks until database is created',
```

Comment:
sure, the old message was `monitor database creation, exit when complete`, which I didn't think was very good at all